outport/firehose/firehoseIndexer.go

```diff
@@ -122,6 +131,57 @@ func (fi *firehoseIndexer) getHeaderBytes(headerHandler data.HeaderHandler) ([]b
 	return headerBytes, headerType, err
 }
 
+func getBody(bodyHandler data.BodyHandler) (*block.Body, error) {
+	if check.IfNil(bodyHandler) {
+		return &block.Body{}, nil
+	}
+
+	body, castOk := bodyHandler.(*block.Body)
+	if !castOk {
+		return nil, errCannotCastBlockBody
+	}
+
+	return body, nil
+}
+
+func getBodyOrNilIfEmpty(body *block.Body) *block.Body {
+	if len(body.MiniBlocks) == 0 {
+		return nil
+	}
+
+	return body
+}
+
+func getAlteredAccounts(accounts map[string]*outportcore.AlteredAccount) []*alteredAccount.AlteredAccount {
+	ret := make([]*alteredAccount.AlteredAccount, 0, len(accounts))
```

Comment:
alteredAccounts instead ret ?

Reply:
Both input and output would have the same name and it would be confusing. 
This applies for all "converting funcs" such as tokens, altered accounts, etc.
Would keep it as it is.